packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -34,9 +33,14 @@ class BottomSheet extends StatelessComponent {
   /// passive observer.
   final Performance performance;
   final VoidCallback onClosing;
-  final double childHeight;
   final WidgetBuilder builder;
 
+  final _childKey = new GlobalKey(debugLabel: 'BottomSheet child');
```

Comment:
This will cause an error if the same instance of BottomSheet is inserted into the tree twice.